ubr/psql_target.py

```diff
@@ -166,19 +166,34 @@ def backup(path_list, destination=None):
         'output': [_backup(dbname, destination) for dbname in path_list if dbexists(dbname)]
     }
 
-def _restore(dbname, backup_dir):
+def backup_missing_prompt_user(dbname, dump_path):
```

Comment:
I know, it's there in `main.py`, just not implemented properly yet. nothing is using the restore process yet in a cron or a highstate.